src/main/js/controls/Timeline/TimelineConfig.js

```diff
@@ -40,6 +40,46 @@ const getPadding = (config, inputPadding) => {
     }
 };
 
+/**
+ * Helper function to set the right legend padding values based on input JSON.
+ *
+ * @private
+ * @param {object} config - config which needs to be updated
+ * @param {object} inputLegendPadding - input legend padding provided via input JSON.
+ * @returns {object} - padding for Legend
+ */
+const getLegendPadding = (config, inputLegendPadding) => {
+    if (utils.isDefined(config.legendPadding)) {
```

Comment:
Can we add a comment in the code here why we are checking the condition?

Reply:
Added comment [here](https://github.com/JananiGunasekaran/carbon-graphs/commit/37d8f2ff39986b1b59f43050174b4319224fda21). This method has been added in GraphConfig.js and GanttConfig.js. So i have added comments in these 2 files too.